client/components/paginate/Thumbnail.tsx

```diff
@@ -1,8 +1,13 @@
 import React, { useEffect, forwardRef } from "react";
-import PropTypes from "prop-types";
 import { usePaginatorContext } from "../../context/PaginatorContext";
 
-const Thumbnail = forwardRef(({ scrollTo, selected, number }, ref) => {
+interface ThumbnailProps {
+    scrollTo: () => void;
+    number: number;
```

Comment:
This is another instance where the orders are transposed (in this case, selected vs. number).

Reply:
Made change.